Firestore/core/src/model/mutable_document.cc

```diff
@@ -113,6 +113,11 @@ std::string MutableDocument::ToString() const {
   return stream.str();
 }
 
+bool MutableDocument::CompareIgnoringState(const MutableDocument& other) const {
```

Comment:
If we keep this, this should be called "EqualsSomethingSomething" as it does not compare documents.